### PR TITLE
Fix issue #41. Clean up coverage and deprecation notices.

### PR DESCRIPTION
**Pull request by rox163**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [x] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG
**Fixed** issue #41. 
**Cleaned** up coverage, demo and deprecation notices.

### CODE REVIEW

File: addon/components/frost-sort-item.js [hunk 1]
```diff
@@ -27,7 +27,6 @@ export default Component.extend(PropTypesMixin, {
 
   getDefaultProps () {
     return {
-      direction: 'asc',
```
Comment (rox163) [inline addon/components/frost-sort-item.js:30]: this was overwriting the first default direction setup by the user.

File: tests/dummy/app/pods/application/template.hbs [hunk 2]
```diff
@@ -1,8 +1,4 @@
-<div class="dummy-notification-container">
-  {{#each notifications as |notification|}}
-    {{notification-message notification=notification}}
-  {{/each}}
-</div>
+{{notification-container position="top-right"}}
```
Comment (rox163) [inline tests/dummy/app/pods/application/template.hbs:1]: fixed notifications for the demo.

File: tests/dummy/app/pods/demo/controller.js [hunk 3]
```diff
@@ -41,6 +43,7 @@ export default Controller.extend({
       this.set('sortOrder', sortItems.map((object) => {
         return object.value + object.direction
       }))
+      console.log(this.get('sortOrder'))
```
Comment (quincyle) [inline tests/dummy/app/pods/demo/controller.js:46]: Woops :)

**Comment by quincyle** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-sort/pull-request/42/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-sort)

**Comment by justinbowes** [comment]
LGTM 👍

**Pull request merged**